Makefile

```diff
@@ -30,6 +30,14 @@ endif
 	echo "$(DOCKER_PASSWORD)" | docker login -u "$(DOCKER_USERNAME)" --password-stdin && \
 	docker push $(IMAGE)
 
+clean:
+	rm -rf sha.txt
```

Comment:
Can we use `version.txt` everywhere instead of `sha.txt`? Someday we may want a different kind of version (e.g., semver).